src/openrct2/network/Network.cpp

```diff
@@ -1470,18 +1471,24 @@ void Network::Server_Send_TOKEN(NetworkConnection& connection)
     connection.QueuePacket(std::move(packet));
 }
 
-void Network::Server_Send_OBJECTS(NetworkConnection& connection, const std::vector<const ObjectRepositoryItem*>& objects) const
+void Network::Server_Send_OBJECTS_LIST(NetworkConnection& connection, const std::vector<const ObjectRepositoryItem*>& objects) const
 {
     log_verbose("Server sends objects list with %u items", objects.size());
-    std::unique_ptr<NetworkPacket> packet(NetworkPacket::Allocate());
-    *packet << static_cast<uint32_t>(NETWORK_COMMAND_OBJECTS_LIST) << static_cast<uint32_t>(objects.size());
-    for (auto object : objects)
+
+    for (size_t i = 0; i < objects.size(); ++i)
```

Comment:
I wonder if isn't clearer if we use ranged-based for and then `std::distance` where the index is clearer.